docs/UsingNetlifyGuide.md

```diff
@@ -0,0 +1,45 @@
+## Using RepoSense with Netlify
+
+[Netlify](https://www.netlify.com) enables you to run RepoSense directly on your target repository. You can do everything on the browser itself and Netlify will handle running RepoSense for you. Therefore, as a user you don't need to download the latest jar file every time when wanting to run RepoSense.
```

Comment:
Therefore, ~as a user~ you don't need to download ~the latest jar file every time when wanting~ anything to run RepoSense.